content/download/package-manager.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Package Manager
+description: Installing Node.js via package manager
+authors: fhemberger, XhmikosR, shadowspawn, vsemozhetbyt, nschonni, wildcard, MrJithil, kasicka, cassidyjames, Trott, richardlau, Qantas94Heavy, pierreneter, 0mp, ThePrez, PoojaDurgad, MaledongGit, Megajin, marc-maurer, yodeyer, geek, sudowork, strawbrary, ryanmurakami, rbnswartz, arkwright, oliversalzburg, mweagle, Mohamed3on, Ginden, kapouer, jperkin, jericopulvera, jedsmith, jasonkarns, sonicdoe, mcollina, fornwall, danbev, naskapal, awochna, AdamMajer, ahmetanilgur, bnb, qbit
+category: download
+---
+
+# Installing Node.js via package manager
+
+***Note:*** The packages on this page are maintained and supported by their respective packagers, **not** the Node.js core team. Please report any issues you encounter to the package maintainer. If it turns out your issue is a bug in Node.js itself, the maintainer will report the issue upstream.
+
+* [Installing Node.js via package manager](#installing-nodejs-via-package-manager)
```

Comment:
This section is already being autogenerated in the `Table Of Contents` section of the page, so I think it could be removed from here.

Reply:
> This section is already being autogenerated in the `Table Of Contents` section of the page, so I think it could be removed from here.

idk how i didnt notice. Thanks! ill make the change now